packages/auth/src/graphql/schema.graphql

```diff
@@ -75,3 +84,14 @@ interface MutationResponse {
   success: Boolean!
   message: String!
 }
+
+enum GrantState {
+  "grant request was created but grant was not approved yet"
+  pending
```

Comment:
There's at least one old PR discussion on the formatting of Graphql enums (I never remember how we capitalize or why)
- https://github.com/interledger/rafiki/pull/254#discussion_r823010225